docs/reference/commandline/attach.md

```diff
@@ -39,6 +39,14 @@ using `CTRL-p CTRL-q` key sequence.
 It is forbidden to redirect the standard input of a `docker attach` command
 while attaching to a tty-enabled container (i.e.: launched with `-t`).
 
+While attach stream is connected to container's stdio, docker uses ~1MB memory 
+buffer to maximize the throughput of the application. If this buffer is reached 
+the speed of the API connection will start to have an effect on the application 
+speed. This is similar to the other applications like SSH. Because of this, it 
```

Comment:
s/applications/transports/, to avoid confusion with "application" in the previous sentence?